CMakeLists.txt

```diff
@@ -43,8 +43,6 @@ set(QTCORE_INCLUDE_DIRS ${Qt5_INCLUDE_DIR} ${Qt5Core_INCLUDE_DIRS})
 set(QTGUI_INCLUDE_DIRS ${Qt5Gui_INCLUDE_DIRS})
 set(QTWIDGETS_INCLUDE_DIRS ${Qt5Widgets_INCLUDE_DIRS})
 
-find_package(expat REQUIRED)
```

Comment:
I don't agree: since expat is a private dependency of HostSupport, the corresponding find_package should definitely be in HostSupport's CMakeLists.txt, and not in the main CMakeLists. As I said in my comment, I also think other find_package (glog, ceres) should be moved to the right place.